src/components/edit/ControlledTextInput.tsx

```diff
@@ -3,14 +3,15 @@ import {
     TextField,
     TextFieldProps,
     Theme,
+    TypographyStyle,
     TypographyVariant,
     useTheme,
-} from "@material-ui/core";
-import grey from "@material-ui/core/colors/grey";
-import { CSSProperties, StyledComponentProps } from "@material-ui/styles";
+} from "@mui/material";
+import { grey } from "@mui/material/colors";
 import React from "react";
+import { MUIStyledProps } from "../../common/styledProps";
 
-interface ControlledTextInputProps extends StyledComponentProps {
+interface ControlledTextInputProps extends MUIStyledProps {
```

Comment:
Any components that are meant to be styleable now need these changes - instead of funneling "classes" prop down, "className" is the thing to funnel and it applies generally, not on a variant basis